src/utils/fetchInfoFromGitHub.js

```diff
@@ -0,0 +1,13 @@
+const fetchInfoFromGitHub = (url, GITHUB_TOKEN) => {
```

Comment:
Don't create a new util file for each function you write. Add this function in an existing utility file under `utils/`. Also define functions using the `function` keyworld and you can define it as:
```js
export function fetchInfoFromGitHub(url, githubToken) {
  ...
}
```

Reply:
Also don't forget to add jsdocs once you rewrite it as a `function`.